### ISSUE TITLE
repo add fails on a new helm setup without running update first

### ISSUE THREAD

**Issue by helgi** [helm/helm-classic#325]
Blew away my helm setup and got myself a new binary et all but didn’t run `update` by accident

 ```
helm repo add deis https://github.com/deis/charts
[ERROR] Could not load /Users/helgi/.helm/config.yaml: open /Users/helgi/.helm/config.yaml: no such file or directory
```

This could show a bit nicer error message or just have certain actions run the eq of `update` if needed.

**Comment by michelleN** [comment]
Yeah, we can definitely run helm doctor to fix the set up before failing this way.

**Comment by mboersma** [comment]
This was fixed by #377. `helm repo add` does the right thing now.
